Extension methods from TimeSeriesStatic.cs (A Typo and A Missing Nuget for Static Time Series)

### System information

- Windows 10 Pro
- .NET 4.6.1

### Issue

- **What did you do?**
I am trying to use extension method `DetectSpikeBySsa` from class `SsaSpikeDetecotStaticExtensions` (there's a typo in the name btw). I'm using static API and TimeSeries package.

- **What happened?**
The extension method (and any other extension methods from that namespace) is not available to use on Scalar\<float\> type.
I am using `SsaSpikeDetection()` test from `TimeSeriesStaticTests.cs` as a reference.

- **What did you expect?**
Expected usage (from the referenced test):
```cs
var staticLearningPipeline = staticData.MakeNewEstimator()
                .Append(r => r.Value.DetectSpikeBySsa(80, ChangeHistoryLength, TrainingWindowSize, SeasonalityWindowSize));
```
but `DetectSpikeBySsa` is not available on my Scalar\<float\>.

### Source code / logs

```cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using Microsoft.ML;
using Microsoft.ML.Data;
using Microsoft.ML.Transforms;
using Microsoft.ML.Transforms.TimeSeries;
using Microsoft.ML.StaticPipe;
using Microsoft.ML.Trainers;

namespace xxx.Modules.Data.ML.Dev
{
    public class DataLoader
    {
        private sealed class SpikePrediction
        {
            [VectorType(3)]
            public double[] Data;
        }

        private class DataObject
        {
            public float Gain { get; set; }
        }

        public DataLoader()
        {
            MLContext mlContext = new MLContext(seed: 0);
            var loader = mlContext.Data.CreateTextLoader(ctx => (
                    Id: ctx.LoadText(0),
                    Begin: ctx.LoadText(1),
                    End: ctx.LoadText(2),
                    Gain: ctx.LoadFloat(3),
                    Confidence: ctx.LoadFloat(4)
                ), separator: ',', hasHeader: true);

            var data = loader.Load("output_intervals.csv");

            var preview = data.AsDynamic.Preview();

            var learningPipeline = loader.MakeNewEstimator()
                .Append(r => (
                    r.Gain.DetectSpikeBySsa(80, 800 / 4, 800 / 2, 800 / 8) // Cannot resolve symbol...
                ));
        }
    }
}
```

Included packages:
```
<packages>
  <package id="Castle.Core" version="4.2.1" targetFramework="net461" />
  <package id="Microsoft.ML" version="1.0.0-preview" targetFramework="net461" />
  <package id="Microsoft.ML.CpuMath" version="1.0.0-preview" targetFramework="net461" />
  <package id="Microsoft.ML.DataView" version="1.0.0-preview" targetFramework="net461" />
  <package id="Microsoft.ML.Experimental" version="0.12.0-preview" targetFramework="net461" />
  <package id="Microsoft.ML.FastTree" version="1.0.0-preview" targetFramework="net461" />
  <package id="Microsoft.ML.ImageAnalytics" version="1.0.0-preview" targetFramework="net461" />
  <package id="Microsoft.ML.Mkl.Redist" version="1.0.0-preview" targetFramework="net461" />
  <package id="Microsoft.ML.Recommender" version="0.12.0-preview" targetFramework="net461" />
  <package id="Microsoft.ML.StaticPipe" version="0.12.0-preview" targetFramework="net461" />
  <package id="Microsoft.ML.TimeSeries" version="0.12.0-preview" targetFramework="net461" />
  <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net461" />
  <package id="NLog" version="4.6.1" targetFramework="net461" />
  <package id="System.Buffers" version="4.4.0" targetFramework="net461" />
  <package id="System.CodeDom" version="4.4.0" targetFramework="net461" />
  <package id="System.Collections.Immutable" version="1.5.0" targetFramework="net461" />
  <package id="System.Drawing.Common" version="4.5.0" targetFramework="net461" />
  <package id="System.Memory" version="4.5.1" targetFramework="net461" />
  <package id="System.Numerics.Vectors" version="4.4.0" targetFramework="net461" />
  <package id="System.Reactive" version="4.0.0" targetFramework="net461" />
  <package id="System.Reflection.Emit.Lightweight" version="4.3.0" targetFramework="net461" />
  <package id="System.Runtime.CompilerServices.Unsafe" version="4.5.0" targetFramework="net461" />
  <package id="System.Threading.Tasks.Dataflow" version="4.8.0" targetFramework="net461" />
  <package id="System.ValueTuple" version="4.4.0" targetFramework="net461" />
  <package id="WampSharp" version="19.3.1" targetFramework="net461" />
</packages>
```

![issue1](https://user-images.githubusercontent.com/5548662/55581510-5ac46800-571d-11e9-9cda-ebf002259f4a.png)
The problem is those methods are not found by IntelliSense. It's like if they weren't in any of the nuget packages that I have.

Compiler error (this is misleading I think):
`1>...\src\Paragon.Modules.Data.ML\Dev\DataLoader.cs(71,35,71,51): error CS7036: There is no argument given that corresponds to the required formal parameter 'trainingWindowSize' of 'TimeSeriesCatalog.DetectSpikeBySsa(TransformsCatalog, string, string, int, int, int, int, AnomalySide, ErrorFunction)'`
When I mouse over it says `Cannot resolve symbol 'DetectSpikeBySsa'`

When I copy and paste this code directly from ML.NET repo, everything works fine and I get the correct results:
```cs
using System.Collections.Generic;
using Microsoft.ML;
using Microsoft.ML.Runtime;
using Microsoft.ML.StaticPipe;
using Microsoft.ML.Transforms.TimeSeries;

/// <summary>
/// Static API extension methods for <see cref="SsaSpikeEstimator"/>.
/// </summary>
public static class SsaSpikeDetecotStaticExtensions
{
    private sealed class OutColumn : Vector<double>
    {
        public PipelineColumn Input { get; }

        public OutColumn(Scalar<float> input,
            int confidence,
            int pvalueHistoryLength,
            int trainingWindowSize,
            int seasonalityWindowSize,
            AnomalySide side,
            ErrorFunction errorFunction)
            : base(new Reconciler(confidence, pvalueHistoryLength, trainingWindowSize, seasonalityWindowSize, side, errorFunction), input)
        {
            Input = input;
        }
    }

    private sealed class Reconciler : EstimatorReconciler
    {
        private readonly int _confidence;
        private readonly int _pvalueHistoryLength;
        private readonly int _trainingWindowSize;
        private readonly int _seasonalityWindowSize;
        private readonly AnomalySide _side;
        private readonly ErrorFunction _errorFunction;

        public Reconciler(
            int confidence,
            int pvalueHistoryLength,
            int trainingWindowSize,
            int seasonalityWindowSize,
            AnomalySide side,
            ErrorFunction errorFunction)
        {
            _confidence = confidence;
            _pvalueHistoryLength = pvalueHistoryLength;
            _trainingWindowSize = trainingWindowSize;
            _seasonalityWindowSize = seasonalityWindowSize;
            _side = side;
            _errorFunction = errorFunction;
        }

        public override IEstimator<ITransformer> Reconcile(IHostEnvironment env,
            PipelineColumn[] toOutput,
            IReadOnlyDictionary<PipelineColumn, string> inputNames,
            IReadOnlyDictionary<PipelineColumn, string> outputNames,
            IReadOnlyCollection<string> usedNames)
        {
            //Contracts.Assert(toOutput.Length == 1);
            var outCol = (OutColumn)toOutput[0];
            return new MLContext().Transforms.DetectSpikeBySsa(
                outputNames[outCol],
                inputNames[outCol.Input],
                _confidence,
                _pvalueHistoryLength,
                _trainingWindowSize,
                _seasonalityWindowSize,
                _side,
                _errorFunction);
        }
    }

    /// <summary>
    /// Perform SSA spike detection over a column of time series data. See <see cref="SsaSpikeEstimator"/>.
    /// </summary>
    public static Vector<double> DetectSpikeBySsa(
        this Scalar<float> input,
        int confidence,
        int changeHistoryLength,
        int trainingWindowSize,
        int seasonalityWindowSize,
        AnomalySide side = AnomalySide.TwoSided,
        ErrorFunction errorFunction = ErrorFunction.SignedDifference
        ) => new OutColumn(input, confidence, changeHistoryLength, trainingWindowSize, seasonalityWindowSize, side, errorFunction);

}
```

--- a/src/Microsoft.ML.TimeSeries.StaticPipe/TimeSeriesStatic.cs
+++ b/src/Microsoft.ML.TimeSeries.StaticPipe/TimeSeriesStatic.cs
@@ -230,7 +230,7 @@ namespace Microsoft.ML.StaticPipe
     /// <summary>
     /// Static API extension methods for <see cref="SsaSpikeEstimator"/>.
     /// </summary>
-    public static class SsaSpikeDetecotStaticExtensions
+    public static class SsaSpikeDetectorStaticExtensions
     {
         private sealed class OutColumn : Vector<double>
         {

